test/unit/lib/population_clone_job_test.rb

```diff
@@ -180,6 +180,19 @@ def test_perform_randomized_races
     assert_races_are_random
   end
 
+  def test_perform_replace_other_race
+    # Clone and ensure they have random races
+    pcj = Cypress::PopulationCloneJob.new('randomize_demographics' => false)
+    prng = Random.new(@pt.rand_seed.to_i)
+    patient = Patient.first
+    # Replace original race code with the code for 'Other'
+    patient.get_data_elements('patient_characteristic', 'race').first.dataElementCodes.first['code'] = '2131-1'
+    pcj.clone_and_save_patient(patient, prng, Provider.first)
+    cloned_patient = Patient.where('extendedData.original_patient' => patient.id).first
+    # Assert that the new race is consistent '2106-3'
+    assert_equal '2106-3', cloned_patient.race
```

Comment:
Is this really a "random" race, if it's always `2106-3`?

Reply:
Your contention is with the comment?  I'll update it.  I copied the basis for the test from the test above it which was for randomness.  This is a test for consistent replacement.

Reply:
Yeah; I'm fine with either updating the comment to match the behavior or vice versa. I just don't want to confuse future readers by having mismatch between comments and behaviors.